kong/cmd/version.lua

```diff
@@ -11,15 +13,45 @@ Options:
 ]]
 
 local str = [[
-Kong: %s
+Kong Core: %s
```

Comment:
As discussed in private, maybe we can introduce the plugins versions in this PR, but maintain the current `kong version` format to avoid a breaking change until 0.12? No strong feelings, just making sure we cover all grounds and we're fine with it.

Reply:
I agree with @thibaultcha: even if we don't say it's safe to, people often run regexes on the output of version commands in their build environments (I know I do!) and it's really annoying when they change formats in arbitrary versions. Let's save this part of the change for 0.12.